api/client/client.go

```diff
@@ -1986,6 +1987,10 @@ func (c *Client) SearchEvents(ctx context.Context, fromUTC, toUTC time.Time, nam
 	for _, rawEvent := range response.Items {
 		event, err := events.FromOneOf(*rawEvent)
 		if err != nil {
+			if trace.IsBadParameter(err) {
+				log.Warnf("skipping unparsable event: %v", err)
```

Comment:
```suggestion
				log.Warnf("skipping unknown event: %v", err)
```

I'm not sure if `unparsable` is a word, but unknown is :-)